tools/tertiary-analysis/scmap/scmap_scmap_cell.xml

```diff
@@ -23,7 +23,7 @@
       <param name="n_nearest_neighbours" type="integer" label="Number of nearest neighbours" value='5' help="A positive integer specifying the number of nearest neighbours to find" />
       
      <conditional name="cluster_projection">
-        <param name="cluster_projection" type="boolean" checked="false" label="Annotate cells of the projection dataset using labels of the reference?" help="If cell cluster annotation is available for the reference datasets, in addition to finding top 10 nearest neighbours scmap-cell also allows to annotate cells of the projection dataset using labels of the reference."/>
+        <param name="do_cluster_projection" type="boolean" checked="false" label="Annotate cells of the projection dataset using labels of the reference?" help="If cell cluster annotation is available for the reference datasets, in addition to finding top 10 nearest neighbours scmap-cell also allows to annotate cells of the projection dataset using labels of the reference."/>
```

Comment:
This fix was integrated previously (right?)- do you have a Git issue?

Reply:
@pinin4fjords, I forgot to update the branch. Should be fixed now.